### DISCUSSION TITLE
Can I Reference TerraGrunt Outputs with "dependency" from another folder/account?

### DISCUSSION THREAD

**Discussion by cxmiller-strib** [gruntwork-io/knowledge-base#805] [How do I]
I have VPCs and subnets created in a networking account and am sharing the subnets with AWS RAM. Is it possible to use ["dependency" blocks](https://terragrunt.gruntwork.io/docs/reference/config-blocks-and-attributes/#dependency) to pull the networking VPC outputs while running TerraGrunt from the stage folder/account?

--networking
  |--region
    |--etc/vpc
--stage
  |--region
    |--ecs <-- Possible to call a dependency here for the VPC outputs?

---

<ins datetime="2024-02-26T17:58:28Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110793">Tracked in ticket #110793</a></p>
</ins>

**Comment by denis256** [comment]
Hi,
I think can be used `iam_role` in `ecs` module but it will require allowing role assumption for used AWS account

https://terragrunt.gruntwork.io/docs/features/work-with-multiple-aws-accounts/
